log/handlers.ts

```diff
@@ -51,6 +52,42 @@ export class BaseHandler {
 }
 
 export class ConsoleHandler extends BaseHandler {
+  format(logRecord: LogRecord): string {
+    if (this.formatter instanceof Function) {
+      return this.formatter(logRecord);
+    }
+
+    let msg = this.formatter.replace(/{(\S+)}/g, (match, p1) => {
+      const value = logRecord[p1];
+
+      // do not interpolate missing values
+      if (!value) {
+        return match;
+      }
+
+      return value;
+    });
+
```

Comment:
No need to copy this logic. You can call `super` first and process return value.